src/service/resources-impl.js

```diff
@@ -124,6 +124,23 @@ export class Resources {
     /** @const {!TaskQueue_} */
     this.queue_ = new TaskQueue_();
 
+    /** @private {boolean} */
```

Comment:
Let's also discuss moving these variables and some of the related code to `performance.js`